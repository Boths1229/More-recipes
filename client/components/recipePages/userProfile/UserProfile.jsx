import React from 'react';
import avatar from '../../../public/images/avatar.png';

const UserProfile = (props) => {
  const { email, username } = props.userData;
  return (
    <div className="row mt-5 pb-3">
      <div className="col-sm-12 profile-bg user-profile">
        <img
          className="img-fluid rounded-circle"
          src={avatar}
          alt="avatar"
        />
        <h3>{username}</h3>
        <p>recipe added: 20
          <span className="vl ml-2 user-text" />
          <span>email: {email}</span>
          <span className="vl ml-2 user-text">favorite recipe: 20 </span>
        </p>
      </div>
    </div>
  );
};

export default UserProfile;
