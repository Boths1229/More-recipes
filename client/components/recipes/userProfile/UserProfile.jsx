import React from 'react';

const UserProfile = props => (
  <div className="row pb-3">
    <div className="col-sm-12 profile-bg user-profile">
      <img className="img-fluid rounded-circle" src="../image/avatar.png" alt="avatar" />
      <h3>Sannikay</h3>
      <p>recipe added: 20
        <span className="vl user-text" />
        <span>email: kensanni@gmail.com </span>
        <span className="vl user-text">favorite recipe: 20</span>
      </p>
    </div>
  </div>
);

export default UserProfile;
