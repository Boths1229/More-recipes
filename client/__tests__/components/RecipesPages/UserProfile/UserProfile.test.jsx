import React from 'react';
import { shallow } from 'enzyme';
import UserProfile from '../../../../components/recipePages/userProfile/UserProfile';

describe('user profile component', () => {
  it('matches snapshot', () => {
    const props = {
      userData: { email: '', username: '' }
    };
    const output = shallow(<UserProfile {...props} />);
    expect(output).toMatchSnapshot();
  });
});
