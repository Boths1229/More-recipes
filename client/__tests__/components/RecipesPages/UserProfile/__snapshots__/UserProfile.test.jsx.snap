// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user profile component matches snapshot 1`] = `
<div
  className="row mt-5 pb-3"
>
  <div
    className="col-sm-12 profile-bg user-profile"
  >
    <img
      alt="avatar"
      className="img-fluid rounded-circle"
      src="test-file-stub"
    />
    <h3 />
    <p>
      recipe added: 
      0
      <span
        className="vl ml-2 user-text"
      />
      <span>
        email: 
      </span>
      <span
        className="vl ml-2 user-text favorite"
      >
        favorite recipe: 
         
      </span>
    </p>
  </div>
</div>
`;
